proofs/cbmc/poly_uniform_gamma1/poly_uniform_gamma1_harness.c

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2025 The mldsa-native project authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "fips202/fips202.h"
```

Comment:
I think this header can be removed.

Reply:
[c0162a8](https://github.com/pq-code-package/mldsa-native/pull/225/commits/c0162a80d93aa245a8e36ea667ad2a417b38d5f6)

Reply:
@jakemas This still seems to be there in the latest version of the PR?